pkg/serviceaccount/tokens_controller_test.go

```diff
@@ -372,7 +372,7 @@ func TestTokenCreation(t *testing.T) {
 	for k, tc := range testcases {
 
 		// Re-seed to reset name generation
-		rand.Seed(1)
+		utilrand.Seed(1)
```

Comment:
I withdraw my comment.  Fixing it is more effort that it is worth.  In an ideal world this coupling would be looser, but it's probably not worth the energy.

Something like api.NameGenerator.Seed(1) which was an instance of our own Rand object from this lib, which was used for all name generation.  That way it was clear what the test was asserting.